source/jobs/prerequisites/oracle.txt

```diff
@@ -136,7 +143,138 @@ Steps
+                  :copyable: true
+
+                  ALTER USER <user> DEFAULT TABLESPACE LOGMINER_TBS QUOTA UNLIMITED ON LOGMINER_TBS CONTAINER=ALL;
+
+   .. tab:: Continuous Jobs (Amazon RDS)
+      :tabid: enable-continuous-jobs-rds
+
+      .. procedure::
+         :style: normal
+
+         .. step:: Set up user permissions
+
+            This procedure applies to Oracle instances which **are 
+            hosted** on Amazon RDS.
+            
+            .. include:: /includes/fact-data-prep-oracle-step1.rst
+
+            d. Grant additional permissions to the service account 
+               to run continuous migration jobs. 
+               
+               Append ``CONTAINER=ALL`` if you're migrating a multi-tenant container database.
+
+               .. code-block:: sql
+                  :copyable: true
+
+                  GRANT FLASHBACK ANY TABLE TO <user>;
+                  GRANT EXECUTE_CATALOG_ROLE TO <user>;
+                  GRANT SELECT ANY TRANSACTION TO <user>;
+                  GRANT LOGMINING TO <user>;
+                  GRANT CREATE TABLE TO <user>;
+                  GRANT LOCK ANY TABLE TO <user>;
+                  GRANT CREATE SEQUENCE TO <user>;
+
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$DATABASE', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('DBA_LOG_GROUPS', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('DBMS_LOGMNR', '<user>', 'EXECUTE');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('DBMS_LOGMNR_D', '<user>', 'EXECUTE');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$LOG', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$LOG_HISTORY', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$LOGMNR_LOGS', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$LOGMNR_CONTENTS', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$LOGMNR_PARAMETERS', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$LOGFILE', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$ARCHIVED_LOG', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$ARCHIVE_DEST_STATUS', '<user>', 'SELECT');
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('V_$TRANSACTION', '<user>', 'SELECT');
+
+               If an RDS instance is multi-tenant, an additional permission is required for tablespaces:
+
+               .. code-block:: sql
+                  :copyable: true
+
+                  CALL rdsadmin.rdsadmin_util.grant_sys_object('DBA_TABLESPACES', '<user>', 'SELECT');
+
+         .. step:: Turn on archive logging
+            
+            a. To examine if archive logging is already enabled, run the following query:
```

Comment:
np: i think `to examine` is a strong word for a mere "true / false" check; my personal preference would be to use `to check` instead